index.bs

```diff
@@ -229,12 +229,11 @@ Their [=deserialization steps=], given |serialized| and |value| are:
   :: Returns the [=entry/name=] of the entry represented by |handle|.
 </div>
 
-The <dfn attribute for=FileSystemHandle>kind</dfn> attribute must
-return {{FileSystemHandleKind/"file"}} if the associated [=FileSystemHandle/entry=] is a [=file entry=],
+The <dfn attribute for=FileSystemHandle>kind</dfn> getter steps are to
+return {{FileSystemHandleKind/"file"}} if this is a [=file entry=],
 and return {{FileSystemHandleKind/"directory"}} otherwise.
```

Comment:
```suggestion
The <dfn attribute for=FileSystemHandle>kind</dfn> getter steps are to return 
{{FileSystemHandleKind/"file"}} if [=this=] is a [=file entry=]; otherwise 
{{FileSystemHandleKind/"directory"}}.
```